docs/design/docs-cli.md

```
@@ -0,0 +1,521 @@
+```
+
+`PUT` returns the new `Document` (echo) so the CLI can emit a confirmation
+payload in JSON.
+
+### 5. Markdown Mapping and Page Slicing
+
+#### 5.1 Markdown Mapping
+
+| Element                                    | Mapping                                                                  |
+| ------------------------------------------ | ------------------------------------------------------------------------ |
+| title                                      | `# `                                                                     |
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)